shell.nix

```diff
@@ -0,0 +1,19 @@
+{pkgs ? import <nixpkgs> {}}: let
```

Comment:
This file allows nix users to automatically have all dependencies in a devshell. If you don't want this included in the repository just ping me and I'll remove it/gitignore it.